messages/de/app.php

```diff
@@ -5,10 +5,10 @@
     'Add Your Server' => 'Füge deinen Server hinzu',
     'Add Bot' => 'Füge unseren Bot hinzu',
     'Add a Tag' => 'Füge ein Stichwort hinzu',
-    'Add <b>CREATE_INSTANT_INVITE</b> and <b>SEND_MESSAGES</b> permissions to DISBOARD bot.' => 'Gebe dem DISBOARD Bot die Berrechtigungen <b>Einladung erstellen</b> und <b>Nachrichten senden</b>.',
+    'Add <b>CREATE_INSTANT_INVITE</b> and <b>SEND_MESSAGES</b> permissions to DISBOARD bot.' => 'Gebe dem DISBOARD Bot die Berechtigungen <b>Einladung erstellen</b> und <b>Nachrichten senden</b>.',
     'Admin response' => 'Antwort eines Administrators',
     'Admin servers' => 'Administrierte Server',
-    'After that send this command: <span class="tag is-light">!d invite</span>' => 'Nachdem du den Command benutzt hast, nutze <span class="tag is-light">!d invite</span>',
```

Comment:
"Nachdem tu den Command benutzt hast" really doesn't fit here as this is the error message on the Dashboard, so I approve the change. I wasn't aware of that when I submitted my translation.

However, "Sende diesen Befehl" sounds weird, so I'm suggesting "**Nutze dann den Befehl !d bump**" or "**Nutze dann diesen Befehl: !d bump**". instead.

Reply:
I'd be for **"Nutze dann den Befehl !d bump"**.